plugin/src/main/java/org/opensearch/ml/utils/MLExceptionUtils.java

```diff
@@ -18,6 +18,8 @@
 public class MLExceptionUtils {
 
     public static final String NOT_SERIALIZABLE_EXCEPTION_WRAPPER = "NotSerializableExceptionWrapper: ";
+    public static final String REMOTE_INFERENCE_DISABLED_ERR_MSG =
```

Comment:
How about : `Remote Inference is currently disabled. To enable it, update the setting "plugins.ml_commons.remote_inference_enabled" to true.`